libs/langchain/langchain/chat_models/anyscale.py

```diff
@@ -99,26 +99,21 @@ def get_available_models(
 
         return {model["id"] for model in models_response.json()["data"]}
 
-    @root_validator(pre=True)
-    def validate_environment_override(cls, values: dict) -> dict:
+    @root_validator()
+    def validate_environment(cls, values: dict) -> dict:
         """Validate that api key and python package exists in environment."""
-        values["openai_api_key"] = get_from_dict_or_env(
-            values,
-            "anyscale_api_key",
-            "ANYSCALE_API_KEY",
-        )
         values["anyscale_api_key"] = convert_to_secret_str(
             get_from_dict_or_env(
                 values,
                 "anyscale_api_key",
                 "ANYSCALE_API_KEY",
             )
         )
-        values["openai_api_base"] = get_from_dict_or_env(
+        values["anyscale_base_url"] = get_from_dict_or_env(
             values,
-            "anyscale_api_base",
-            "ANYSCALE_API_BASE",
-            default=DEFAULT_API_BASE,
+            "anyscale_base_url",
+            "ANYSCALE_BASE_URL",
```

Comment:
could you revert this name change? This will be a breaking change to anyone currently using this class.

Reply:
agreed. revert it back